examples/meta/generator/translate.py

```diff
@@ -483,7 +483,7 @@ def translateExpr(self, expr):
             method = expr[key][0]["Identifier"]
             argsList = None
             try:
-                argsList = expr[key][2]
+                argsList = expr[key][1]
```

Comment:
Hi @karlnapf ,   I already fix the missing parameter with global function call in here. Thank you for your help :)